constellations/orion/src/repos/migrations/000017_add_position_achievements.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE achievements
```

Comment:
17 is now taken up by the google classroom code. I think this should be file 19 now. Same with the `.up.sql` file.